architecture/core_concepts/pods_and_services.adoc

```diff
@@ -159,6 +160,51 @@ link:https://kubernetes.io/docs/concepts/workloads/pods/pod/[Kubernetes pod docu
 has details about the functionality and purpose of pods.
 ====
 
+[[admin-manage-pod-restart]]
+=== Pod Restart Policy
+//from https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy
+A pod restart policy determines how {product-title} responds when containers in a pod exit.
```

Comment:
I think there is a word missing here.  Is it supposed to be 'when containers are in a pod exit'?

Reply:
_containers in a pod_ and _containers in the pod_ seem to be common phrases, based on a search in Atom. It is like _peas in a pod_!